roles/ceph-dashboard/tasks/configure_dashboard.yml

```diff
@@ -10,7 +10,7 @@
   block:
    - name: get SSL status for dashboard
      run_once: true
-     command: "{{ container_exec_cmd }} ceph --cluster {{ cluster }} config get mgr mgr/dashboard/ssl"
+     command: "{{ container_exec_cmd }} ceph --cluster {{ cluster }} config get mgr.{{ hostvars[groups[mgr_group_name][0]]['ansible_hostname'] }} mgr/dashboard/ssl"
```

Comment:
We probably need to do some conditioning here if the mgr group isn't defined and fallback to mons[0] (implicite mgr on mon)